### DISCUSSION TITLE
How to force clean the build cache with gatsby?

### DISCUSSION THREAD

**Discussion by theowenyoung** [vercel/vercel#4152] [General] [answered]
HI, I use gatsby for site, and zeit auto cache `.cache` and `public`, how ever some wrong data are cached, I need to clean all cache to build fresh, how can I do?

Thanks for any help

**Comment by theowenyoung** [comment]
Netlify offer a `force clean cache and deploy` option

**Comment by lucleray** [comment, marked as answer]
Hi @theowenyoung,

If you are using a Git Integration to deploy to Vercel, you can clear the cache by using the "Redeploy" button in the deployment page of the dashboard:
![image](https://user-images.githubusercontent.com/6616955/80288107-00d27e00-8736-11ea-90b9-cc3d506e768c.png)

A new deployment will be created with a cleared cache.

If you are using our CLI to deploy, you can pass the `--force` flag like this:
```
now --force
```

**Reply by theowenyoung** [reply]
Get it, Thanks~
